bert.py

```diff
@@ -235,17 +235,14 @@ def encode(self, hidden_states, attention_mask):
         # B, T, C
         return hidden_states
 
-    def forward(self, input_ids, attention_mask):
+    def forward_with_embedding(self, embedding, attention_mask):
```

Comment:
This is needed because we need to perturb the embedding and forward it (instead of input_id)